src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -480,4 +480,10 @@ Use the dropdown to view and switch to other projects this file may belong to.</
   <data name="WorkspaceOutputPaneTitle" xml:space="preserve">
     <value>IntelliSense</value>
   </data>
+  <data name="WRN_MissingAnalyzerReferenceId" xml:space="preserve">
+    <value>MissingAnalyzerReference</value>
```

Comment:
Do we need to localize the ID? Probably yes given its a description of the symptom.